src_client/Window.re

```diff
@@ -0,0 +1,8 @@
+// Some bindings to functions available in the browser global `window` object
```

Comment:
maybe it would be easier to use bs-fetch?

Reply:
I think it would be easier to use bs-fetch, but on the other hand, maybe we want to keep a binding example in there for beginners. If it looks like workshop will run too long, this is something we could cut.

Reply:
I'm fine with both. I agree it could be good for the attendants to see some binding more directly, if they're curious.